packages/lesswrong/components/posts/PostsPage/LWPostsPageHeader.tsx

```diff
@@ -138,7 +138,7 @@ const LWPostsPageHeader = ({post, showEmbeddedPlayer, toggleEmbeddedPlayer, clas
 
   const linkpostDomain = post.url && new URL(post.url).hostname;
   const linkpostTooltip = <div>This is a linkpost:<br/>{post.url}</div>;
-  const linkpostNode = post.url && feedDomain !== linkpostDomain ? <LWTooltip title={linkpostTooltip}>{linkpostDomain}</LWTooltip> : null;
+  const linkpostNode = post.url && feedDomain !== linkpostDomain ? <LWTooltip title={linkpostTooltip}><a href={post.url} target="_blank" rel="noopener noreferrer">{linkpostDomain}</a></LWTooltip> : null;
```

Comment:
Not sure why those values for target or rel - they don't match up to what's in `LinkPostMessage` (which is what I think it used to be) - but if it's deliberate, then 👍

Reply:
Yeah the first pass of this was LLM-generated and wrong, but I figured it out on my own.